Library/Formula/puddletag.rb

```diff
@@ -45,7 +38,40 @@ def install
     ENV.prepend_create_path "PYTHONPATH", HOMEBREW_PREFIX/"lib/python2.7/site-packages"
 
     bin.install Dir[libexec/"bin/*"]
-    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    bin.env_script_all_files(libexec/"bin", :PATH => HOMEBREW_PREFIX/"bin:$PATH", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    Pathname("Info.plist").write <<-EOS.undent
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+      <dict>
+        <key>CFBundleDevelopmentRegion</key>
+        <string>English</string>
+        <key>CFBundleExecutable</key>
+        <string>Puddletag</string>
+        <key>CFBundleIconFile</key>
+        <string>Puddletag</string>
+        <key>CFBundleIdentifier</key>
+        <string>com.github.keithgg.puddletag</string>
+        <key>CFBundleInfoDictionaryVersion</key>
+        <string>6.0</string>
+        <key>CFBundlePackageType</key>
+        <string>APPL</string>
+        <key>CFBundleVersion</key>
+        <string>1.0</string>
+        <key>CFBundleSignature</key>
+        <string>PUDDLE</string>
+      </dict>
+      </plist>
+    EOS
+
+    mkdir_p "Puddletag.app/Contents/MacOS"
+    mkdir_p "Puddletag.app/Contents/Resources"
+    cp buildpath/"Info.plist", "Puddletag.app/Contents"
+    system "sips", "-s", "format", "icns", buildpath/"puddletag.png", "--out", "Puddletag.app/Contents/Resources/Puddletag.icns"
+    cp prefix/"bin/puddletag", "Puddletag.app/Contents/MacOS/Puddletag"
+    chmod "+x", "Puddletag.app/Contents/MacOS/Puddletag"
+    prefix.install "Puddletag.app"
```

Comment:
I totally understand.
The app bundle I wrote is more or less a "windows link" not the app itself. It just contains an icon and the wrapper script made by homebrew for starting. Nothing else, no libs, not anything else.
Is there a mechanism in hombrew which would provide the thing I need?